### PR TITLE
Add csv headers.

### PR DESCRIPTION
**Pull request by ignaloidas**

### CODE REVIEW

**Comment by robertasstankevicius** [comment]
Update HISTORY.md

**Comment by ignaloidas** [comment]
> Update HISTORY.md

Done

**Review by robertasstankevicius — APPROVED** [review]

File: b_lambda_layer_common/source/python/b_lambda_layer_common/api_gateway/response_headers.py [hunk 1]
```diff
@@ -56,3 +56,13 @@ def mpeg_headers() -> 'ResponseHeaders':
             # Indicate that response is of audio type, mpeg subtype.
             'Content-Type': 'audio/mpeg'
         })
+
+    @staticmethod
+    def csv_headers() -> 'ResponseHeaders':
+        """
+        Audio file WAV content type header.
```
Comment (laimonassutkus) [inline b_lambda_layer_common/source/python/b_lambda_layer_common/api_gateway/response_headers.py:63]: Bad docstring.

File: b_lambda_layer_common/source/python/b_lambda_layer_common/api_gateway/response_headers.py [hunk 2]
```diff
@@ -56,3 +56,13 @@ def mpeg_headers() -> 'ResponseHeaders':
...
+        """
+        return ResponseHeaders({
+            # Indicate that response is of audio type, wav subtype.
```
Comment (laimonassutkus) [inline b_lambda_layer_common/source/python/b_lambda_layer_common/api_gateway/response_headers.py:66]: Bad comment.

**Review by laimonassutkus — CHANGES_REQUESTED** [review]

**Review by laimonassutkus — APPROVED** [review]

**Pull request merged**